cdp_scrapers/legistar_utils.py

```diff
@@ -912,11 +931,10 @@ def filter_event_minutes(
         """
         if not ev_minutes_item.minutes_item or not ev_minutes_item.minutes_item.name:
             return ev_minutes_item
-
         for filter in self.IGNORED_MINUTE_ITEMS:
-            if filter.lower() in ev_minutes_item.minutes_item.name.lower():
+            filterPattern = re.compile(filter.lower())
+            if filterPattern.search(ev_minutes_item.minutes_item.name.lower()):
```

Comment:
This is a nitpick. We don't have regex patterns in `IGNORED_MINUTE_ITEMS`. So using `re.search()` seems unnecessary to me.

I do agree it is probably more future-proof to use `re.search()`. If so, then we should take advantage of the `re` module. e.g.

```python
for filter in self.IGNORED_MINUTE_ITEMS:
    if re.search(filter, ev_minutes_item.minutes_item.name, re.IGNORECASE):
        ...
```

Also, using `re.compile()`, to me, makes sense only if we reuse the object. e.g.
```python
def __init__(self, ...):
    ...
    self.IGNORED_MINUTE_ITEMS: List[Pattern] = [
        re.compile("This meeting also constitutes a meeting of the City Council"),
        ...
    ]
    ...
```

Reply:
I did  re.search because kingcounty minute_item name sometimes have multiline strings which when added to ignored_items were not properly removed/ignored. so I just kept a part of string in the list of ignored items to search I believe linting will complain if I keep teh entire multiline string in one line and when changed to multiline it was not getting picked up during filtering. The only reason I added pattern because ignored_items for multiline strings was not working expected. Let me know if you have any suggestions.

Reply:
By multiline strings do you mean:

```python
"some text\nsome other text"
```

or

```python
(
    "some text "
    "some other text"
)
```

I.e. the first one is an actual multiline string because of `\n` newline character and the second one is a single line of text split onto two lines in the code (useful for code legibility and linting / formatting)

Reply:
@JacksonMaxfield : One of the minute_items have a name like below
name='HOW TO WATCH/LISTEN TO THE MEETING There are three ways to watch or listen to the meeting 1 Stream online via this link https://livestream.com/accounts/15175343/events/4485487 or input the link web address into your web browser 2 Watch King County TV Channel 22 (Comcast Channel 22 and 322(HD Wave Broadband Channel 22 3\tListen to the meeting by telephone Dial 1 253 215 8782 Meeting ID 937 1830 5122 Password 101895 To help us manage the meeting please use the Livestream or King County TV options listed above if possible to watch or listen to the meeting.'

when I added to ignored_items as a multiline string , it didn't get filtered. That's why I added a small text from this  large string and used pattern technique to match and remove them.

Reply:
Great. That makes sense to me.

Reply:
I agree with @dphoria the name of the `IGNORE_MINUTES_ITEM` variable should be changed to `IGNORE_MINUTES_ITEM_PATTERNS` because we are using regex matching.